pages/services/edge-lb/1.0/installing/index.md

```diff
@@ -3,34 +3,133 @@ layout: layout.pug
 ---
 
 Configure a service account and install the Edge-LB package using the instructions below.
 
 **Prerequisites:**
 
-- [DC/OS CLI installed](/latest/cli/install/) and be logged in as a superuser.
-- The [DC/OS Enterprise CLI installed](https://docs.mesosphere.com/1.10/cli/enterprise-cli/).
-- Access to [the remote Edge-LB repositories](https://support.mesosphere.com/hc/en-us/articles/213198586).
+- [DC/OS CLI is installed](/latest/cli/install/)
+- You are logged in as a superuser
+- The [DC/OS Enterprise CLI is installed](https://docs.mesosphere.com/1.10/cli/enterprise-cli/).
+- You have access to [the remote Edge-LB repositories](https://support.mesosphere.com/hc/en-us/articles/213198586).
 
 **Limitations**
 - Currently, Edge-LB works only with DC/OS Enterprise in permissive mode on DC/OS 1.10, and permissive or strict mode on DC/OS 1.11 [security mode](/latest/security/ent/#security-modes). It does not work with disabled mode.
 
 # Add Edge-LB package repositories
 The Edge-LB package is composed of two components: the Edge-LB API server and the Edge-LB pools. You must install universe repos for the Edge-LB API server and the Edge-LB pool in order to install Edge-LB. The Edge-LB API server is a restful API that manages one or more Edge-LB pools. Each Edge-LB pool is a collection of load balancers. An Edge-LB pool can be used to launch one or more instances of a load balancer to create a single highly available load balancer. Currently the Edge-LB pool supports only HAProxy as a load balancer.
 
-Download the artifacts for each of the repos from the [Mesosphere support page](https://support.mesosphere.com/hc/en-us/articles/213198586).
+**Note** If your environment is behind a firewall or otherwise not able to access the public catalog, then you must use a local catalog.
+
+1. Download the artifacts for each of the repos from the [Mesosphere support page](https://support.mesosphere.com/hc/en-us/articles/213198586).
+
+**Note:** You must have a service account to do this.
 
-Once you have the links to the artifacts for the Edge-LB API server and Edge-LB pool repos, use the following command to add them to the universe package repository:
+2. Once you have the links to the artifacts for the Edge-LB API server and Edge-LB pool repos, use the following command to add them to the universe package repository:
 
 ```bash
 dcos package repo add --index=0 edgelb-aws \
-  https://<AWS S3 bucket>/stub-universe-edgelb.json
+  https://<insert download link>/stub-universe-edgelb.json
+```
+
+```bash
 dcos package repo add --index=0 edgelb-pool-aws \
-  https://<AWS S3 bucket>/stub-universe-edgelb-pool.json
+  https://<insert download link>/{{stub-universe-edgelb-pool.json}}
+```
+
+[enterprise]
+# Build your own local Universe
+[/enterprise]
+
+## Adding Stub Universes (Custom Universe Packages)
+If you've been provided stub universe json files (such as for Edge-LB), you can add them to the local universe with the `add-stub-universe.sh` script.
```

Comment:
As I already mentioned before, the customer will probably not know where to look for this script.